### PR TITLE
Flink pipe

### PR DESCRIPTION
**Pull request by viktorria**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1 +1,2 @@
-**/target/**/*
\ No newline at end of file
+**/target/**/*
```
Comment (majercakdavid) [inline .gitignore]: What is the difference? Why did this file change?

File: flink-processing/flink-processing/src/main/scala/Main.scala [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+import org.apache.flink.streaming.api.scala._
+
+object Main {
+  def main(args1: Array[String]): Unit = {
+
+    val env = StreamExecutionEnvironment.getExecutionEnvironment
+    env.getConfig.disableClosureCleaner()
+
+    val kafkaConsumer = new KafkaFlinkConsumer("tweets")
+    val kafkaProducer = new KafkaFlinkProducer("tweets-analysed")
```
Comment (majercakdavid) [inline flink-processing/flink-processing/src/main/scala/Main.scala]: Will not it be results? Should not it be something like tweets-results/tweets-processed?

File: flink-processing/flink-processing/src/main/scala/KafkaFlinkConsumer.scala [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+import org.apache.flink.streaming.api.scala._
+import org.apache.flink.api.common.serialization.SimpleStringSchema
+import org.apache.flink.streaming.connectors.kafka.FlinkKafkaConsumer
+
+class KafkaFlinkConsumer(topic: String) {
+
+  def getProps() = {
+    val properties = new Properties()
+    properties.setProperty("zookeeper.connect", "localhost:2181")
```
Comment (majercakdavid) [inline flink-processing/flink-processing/src/main/scala/KafkaFlinkConsumer.scala:12]: Read it from environment do not make it hard-coded.

File: flink-processing/flink-processing/src/main/scala/KafkaFlinkProducer.scala [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+
+import org.apache.flink.streaming.api.scala._
+import org.apache.flink.api.common.serialization.SimpleStringSchema
+import org.apache.flink.streaming.connectors.kafka.FlinkKafkaProducer
+
+class KafkaFlinkProducer(topic: String) {
+
+  def getProps() = {
+    val properties = new Properties()
+    properties.setProperty("zookeeper.connect", "localhost:2181")
```
Comment (majercakdavid) [inline flink-processing/flink-processing/src/main/scala/KafkaFlinkProducer.scala:12]: Just like the one above

**Review by majercakdavid — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (viktorria) [reply .gitignore]: It is not in the changed files. It comes from merging your branch into mine.

File: flink-processing/flink-processing/src/main/scala/Main.scala (hunk 2 above)
Comment (viktorria) [reply flink-processing/flink-processing/src/main/scala/Main.scala]: I wanted to have a new channel that describes what was done with tweets. And we have analysed their sentiment, so they will be analysed. There is no result in this case.

File: flink-processing/flink-processing/src/main/scala/KafkaFlinkConsumer.scala (hunk 3 above)
Comment (viktorria) [reply flink-processing/flink-processing/src/main/scala/KafkaFlinkConsumer.scala:12]: I have followed project twitter-kafka-scala where properties for producer/consumer are also hard-coded. We can make a new branch to add environments and change it in all projects?

Comment (viktorria) [reply flink-processing/flink-processing/src/main/scala/KafkaFlinkConsumer.scala:12]: Or do you prefer to change it here? before merge?

File: flink-processing/flink-processing/src/main/scala/Main.scala (hunk 2 above)
Comment (majercakdavid) [reply flink-processing/flink-processing/src/main/scala/Main.scala]: The job has side effects on the passed in data and therefore the analysed word is misleading

**Pull request merged**